packages/esm-patient-labs-app/src/config-schema.ts

```diff
@@ -41,16 +41,53 @@ export const configSchema = {
       _default: '52a447d3-a64a-11e3-9aeb-50e549534c5e',
     },
   },
+  labTests: {
+    _type: Type.Array,
+    _elements: {
+      labTestUuid: {
+        _type: Type.UUID,
+        _description: "UUID for the 'Lab' Test that require order reason",
+        _default: '',
+      },
+      labTestOrderReasons: {
+        _type: Type.Array,
+        _elements: {
+          uuid: {
+            _type: Type.UUID,
+            _description: 'Concept UUID representing the order reason',
+          },
+          label: {
+            _type: Type.String,
+            _default: null,
+            _description: 'The label for order reason',
+          },
+        },
+        _default: [],
+        _description: 'Coded Lab test order reason options',
+      },
+    },
+    _default: [],
+    _description: 'Whether to allow for provision of coded order reason',
```

Comment:
Not sure this accurately communicates what this config property does. It looks like we have an array that contains concepts for lab test orders that require a reason for ordering whose structure is something like:

```ts
// labTests array 
[ 
  {
    "labTestUuid": "52a447d3-a64a-11e3-9aeb-50e549534c5e",
    "labTestOrderReasons": [
      {
        "uuid": "52a447d3-a64a-11e3-9aeb-50e549534c5e",
        "label": "Test Reason 1"
      },
      {
        "uuid": "52a447d3-a64a-11e3-9aeb-50e549534c5e",
        "label": "Test Reason 2"
      }
    ] 
  }
]
```

Perhaps the config property would be better named `labTestOrdersWithReasonsForOrdering`.